classes/Provider/Gateways/WebGatewayProvider.php

```diff
@@ -67,12 +74,18 @@ public function boot(Application $app)
         $web->get('/dashboard', 'OpenCFP\Http\Controller\DashboardController::showSpeakerProfile')->bind('dashboard');
 
         // Talks
-        $web->get('/talk/edit/{id}', 'OpenCFP\Http\Controller\TalkController::editAction')->bind('talk_edit')->before($asSpeaker);
-        $web->get('/talk/create', 'OpenCFP\Http\Controller\TalkController::createAction')->bind('talk_new')->before($asSpeaker);
-        $web->post('/talk/create', 'OpenCFP\Http\Controller\TalkController::processCreateAction')->bind('talk_create')->before($asSpeaker);
-        $web->post('/talk/update', 'OpenCFP\Http\Controller\TalkController::updateAction')->bind('talk_update')->before($asSpeaker);
-        $web->post('/talk/delete', 'OpenCFP\Http\Controller\TalkController::deleteAction')->bind('talk_delete')->before($asSpeaker);
-        $web->get('/talk/{id}', 'OpenCFP\Http\Controller\TalkController::viewAction')->bind('talk_view')->before($asSpeaker);
+        $web->get('/talk/edit/{id}', 'OpenCFP\Http\Controller\TalkController::editAction')
+            ->bind('talk_edit')->before($asSpeaker)->before($csrfChecker);
```

Comment:
If we start wrapping, can we wrap all method calls?

Reply:
I wrapped them all

Reply:
Ha, I meant more like wrap all of the *chained* method calls so they will always be on a new line, like this:

```php
$web->get('/talk/edit/{id}', 'OpenCFP\Http\Controller\TalkController::editAction')
    ->bind('talk_edit')
    ->before($asSpeaker)
    ->before($csrfChecker);
```

But, whatever @chartjes and @mdwheele prefer!